### ISSUE TITLE
Discussion: Supporting accessibility names should be more easy

### ISSUE THREAD

**Issue by ArchieCoder** [microsoft/microsoft-ui-xaml#1163]
# Discussion: Supporting accessibility names should be more easy
Right now, to add accessibility description, the syntax is as follow ControlName.[using:Windows.UI.Xaml.Automation]AutomationProperties. This syntax is not user friendly. There is no easy way to discover this syntax unless you consult the documentation. I could extend this problem with localization of control properties.

Suggestion: the resource editor can provide a property helper window where a developer can select a property and then the value. Intellisense can also help.
